srcpkgs/ly/files/ly/conf

```diff
@@ -0,0 +1,8 @@
+BAUD_RATE=38400
+TERM_NAME=linux
+# don't change the following value without also changing the "tty" value in /etc/ly/config.ini
+TTY=7
+
+if [ "${TTY}" = "1" ]; then
+	GETTY_ARGS="--noclear"
+fi
```

Comment:
What do you think about adding an option to source `/etc/locale.conf` ? Having `LANG` set can be nice for your session.

Reply:
When you say "option", do you mean putting it in an `if` block like the `GETTY_ARGS`, or having it be commented out by default in the `conf` file with an explanation of what it will do if un-commented?

Reply:
"option" is a bad word. You can just check if it exists, and if so, source it. Kinda like it's done for the `conf` file itself.

Reply:
I've added that in the most recent push.